node-process-argv/process.js

```diff
@@ -0,0 +1,2 @@
+const process = require('process');
```

Comment:
`process` is available globally so it isn't necessary to require it in.

Reply:
copy that, appreciate the clarification!